Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,41 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            bool again = true;
```

Comment:
Logical variables should be prefixed with is/should/are/was and so on.
``isAgain`` is a bit better for a name.